RELEASE/scripts/autoscend/auto_quest_level_12.ash

```diff
@@ -1290,6 +1299,223 @@ boolean L12_themtharHills()
 	return true;
 }
 
+boolean L12_farm()
+{
+	if(get_property("auto_skipL12Farm").to_boolean())
+	{
+		return false;
+	}
+	if(get_property("sidequestFarmCompleted") != "none")
+	{
+		set_property("auto_skipL12Farm", "true");
+		return false;
+	}
+	if(internalQuestStatus("questL12War") != 1)
+	{
+		return false;
+	}
+	
+	string faction = "fratboy";
+	if(get_property("auto_hippyInstead").to_boolean())
```

Comment:
Instead of `string faction`, either use `get_property("auto_hippyInstead").to_boolean()` like the rest of the code in this file or make a utility function `string auto_warSide()` or `boolean auto_hippyInstead()`

The reason for this is so that someone reading this function doesn't get confused as to why it's using a `faction` variable instead of checking the properties directly - "is there a case where the two are not the same?" they might wonder

Reply:
The reason for the conversion is the mafia sidequests completed properties.
They all use a string with 3 values:
"none" means the sidequest is not complete
"fratboy" means the sidequest was completed for the fratboys
"hippy" means the sidequest was completed for the hippies

So I needed to check things like
`if(get_property("sidequestArenaCompleted") == faction)`
and trying to put 
`get_property("auto_hippyInstead").to_boolean()`
in it will not work.

After I created that string I thought to myself "hey, since I am already going to need this string variable for some of the stuff I am doing, why not also use it to see which side i am on? it is also much clearer to read".

However, if you think it is confusing I don't mind switching it. I will change it out for auto_warSide()

Reply:
commit 3be6d3e